src/main/java/seedu/address/model/event/StartTime.java

```diff
@@ -3,55 +3,55 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
+import java.text.DecimalFormat;
+import java.time.DateTimeException;
 import java.time.LocalTime;
-import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeParseException;
+
 
 /**
  * Represents an Event's starting time in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidStartTime(String)}
  */
 public class StartTime implements Comparable<StartTime> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Start time must be in format: hh/mm/AM, hh/mm/PM or hh:mm";
-
-    //for checking if valid input date format
-    private static final DateTimeFormatter checkFormatter = DateTimeFormatter
-            .ofPattern("[HH:MM][hh/mm/a][h/mm/a][hh/m/a][h/m/a]");
-
-    //for changing to storage friendly format
-    private static final DateTimeFormatter logFormatter = DateTimeFormatter.ofPattern("hh/mm/a");
-
-    //for changing to user-readable format
-    private static final DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("hh:mm a");
-
+    public static final String MESSAGE_FORMAT_CONSTRAINTS = "Start time must be in format: hh:mm.";
```

Comment:
I think omit the fullstop here so as to not confuse the reader

```suggestion
    public static final String MESSAGE_FORMAT_CONSTRAINTS = "Start time must be in format: hh:mm";
```